### ISSUE TITLE
Enabling the add-on causes NVDA to lag heavily in VLC

### ISSUE THREAD

**Issue by cary-rowen** [paulber19/VLCAccessEnhancementNVDAAddon#2]
Hi paul,

You can barely navigate in VLC with NVDA, you have to wait a long time for each key press. For example, to open the menu bar.

* nvda: alpha-27156,26c503ea （2023.1.0.27156）
* VLC:  3.0.17.4 Vetinari

**Comment by paulber19** [comment]
Hi Rowen.
I don't see any slowdown with my setup.
Can you give me more details and privately send me a log of NVDA in 
debug mode.
Thanks.
Best regards.
Paul.

**Comment by cary-rowen** [comment]
Hello, I created a portable version of NVDA separately and then grabbed the log, it does not have private information, I can directly provide it here.
[nvda.log](https://github.com/paulber19/VLCAccessEnhancementNVDAAddon/files/10007224/nvda.log)

**Comment by paulber19** [comment]
Hi.
Thanks for the log.
What happens is normal because the vlc language you are using is Chinese 
(zh_cn).
But for the add-on to work, it must recognize this language.
To do this, a strings-zh_cn.ini file must be in the vlcLocale folder 
with the information corresponding to the language of vlc.
It is in principle the translator who sets it up.
But this add-on has not yet been translated into Chinese.
I still noted oddities in the journal that I will analyze.
Best regards.

**Comment by cary-rowen** [comment]
Thanks @paulber19 for the quick response,
If I have time, I will provide Simplified Chinese translation.

Cheers
